microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```diff
@@ -42,7 +42,7 @@ This article describes the process for cross-tenant mailbox moves and provides g
 
 ## Licensing
 
-Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (onetime fee). Please contact your Microsoft account team for details.
+Cross Tenant User Data Migration is available as an add-on to the following Microsoft 365 subscription plans for Enterprise Agreement customers. User licenses are per migration (one-time fee) and can be assigned either on the source or target user object. This license also covers OneDrive for Business migration. Please contact your Microsoft account team for details.
```

Comment:
Update this "This license also covers OneDrive for Business migration link to content" to add the link to the documentation for this https://learn.microsoft.com/microsoft-365/enterprise/cross-tenant-onedrive-migration

Reply:
done